remote/rc.xml

```diff
@@ -52,7 +52,7 @@
 
 <theme>
   <name>Clearlooks</name>
```

Comment:
```suggestion
  <name>Natura</name>
```

Clearlooks looks dated, esp. with the new border adjustments. `Nature` looks more modern and is shipped as default theme.